bucket/fvim.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.2.273",
+    "description": "Cross platform Neovim front-end UI, built with F# + Avalonia",
+    "homepage": "https://github.com/yatli/fvim",
+    "license": "MIT",
```

Comment:
```suggestion
    "license": "MIT",
    "suggest": {
        "neovim": "extras/neovim"
    },
```

Reply:
I've made the 'Neovim' on line 7 all lowercase